alf/algorithms/tasac_algorithm.py

```diff
@@ -49,8 +47,10 @@
     default_value=())
 
 TasacInfo = namedtuple(
-    "TasacInfo",
-    ["action_distribution", "action", "actor", "critic", "alpha", "repeats"],
+    "TasacInfo", [
+        "reward", "step_type", "discount", "action", "action_distribution",
+        "rollout_beta", "beta", "actor", "critic", "alpha", "repeats"
```

Comment:
rollout_beta -> rollout_b; beta -> b (beta is the policy name and b is the output)

Reply:
updated.